crates/boundless-market/src/contracts/boundless_market.rs

```diff
@@ -1154,7 +1155,7 @@ impl<P: Provider> BoundlessMarketService<P> {
         self.query_request_submitted_event(request_id, None, None).await
     }
 
-    /// Returns journal and seal if the request is fulfilled.
+    /// Returns callback data and seal if the request is fulfilled.
```

Comment:
I'm not 100% convinced this naming makes sense here, how about `claim_data` or something else but not necessarily related to the callback concept